### PR TITLE
Configure Rollbar for crash reporting

### PR DESCRIPTION
**Pull request by pablog**
## Summary ##

Integrated `Rollbar` service for crash reporting.

It reports the `environment` in which the crash took place.

## Known issues

It won't report the stack trace, since for this the `.dsym` file is necessary. There's a fastlane script that does this for us (in a future PR).

Need to add the documentation for Rollbar in [iOS kickoff guide](https://github.com/Wolox/tech-guides/blob/master/iOS/docs/kickoff/README.md)

### CODE REVIEW

**Comment by woloxci** [comment]
**CodeStats metrics details:**

:white_check_mark: Code Quality (Swiftlint): 100.0


Click [here](http://codestats.wolox.com.ar/organizations/wolox/projects/ios-base-project/branches/integrate-rollbar) for more details.

**Comment by woloxci** [comment]
**CodeStats metrics details:**

:white_check_mark: Code Coverage (Slather): 0.0
:white_check_mark: Code Quality (Swiftlint): 100.0


Click [here](http://codestats.wolox.com.ar/organizations/wolox/projects/ios-base-project/branches/integrate-rollbar) for more details.

File: BaseProject/Services/RollbarService.swift [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+//
+//  Created by Pablo Giorgi on 5/18/17.
+//  Copyright © 2017 Wolox. All rights reserved.
+//
+
+import Rollbar
+import Core
+import ReactiveSwift
+
+internal protocol RollbarServiceType {
```
Comment (damfinkel) [inline BaseProject/Services/RollbarService.swift]: why do you need this? If you do, I'd write the {} in the same line as the declaration

Comment (pablog) [inline BaseProject/Services/RollbarService.swift]: I don't need it. I removed it.

**Comment by woloxci** [comment]
**CodeStats metrics details:**

:white_check_mark: Code Coverage (Slather): 0.0
:white_check_mark: Code Quality (Swiftlint): 100.0


Click [here](http://codestats.wolox.com.ar/organizations/wolox/projects/ios-base-project/branches/integrate-rollbar) for more details.

**Comment by pablog** [comment]
Sure @danielaRiesgo https://github.com/Wolox/tech-guides/issues/61

**Comment by woloxci** [comment]
**CodeStats metrics details:**

:white_check_mark: Code Coverage (Slather): 0.0
:white_check_mark: Code Quality (Swiftlint): 100.0


Click [here](http://codestats.wolox.com.ar/organizations/wolox/projects/ios-base-project/branches/integrate-rollbar) for more details.

**Comment by woloxci** [comment]
**CodeStats metrics details:**

:white_check_mark: Code Coverage (Slather): 0.0
:white_check_mark: Code Quality (Swiftlint): 100.0


Click [here](http://codestats.wolox.com.ar/organizations/wolox/projects/ios-base-project/branches/integrate-rollbar) for more details.

**Pull request merged**